src/components/Reactions/Reactions.scss

```diff
@@ -0,0 +1,49 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}reactions';
+
+#{$block} {
+    &__reaction-button_add-button {
+        color: var(--g-color-text-secondary);
+    }
+
+    &__popup {
+        padding: 8px;
+    }
+
+    &__add-reaction-popover {
+        max-width: unset;
+    }
+
+    &__reaction-button_size_xs {
+        font-size: 12px;
+    }
+    &__reaction-button_size_s {
+        font-size: 16px;
+    }
+    &__reaction-button_size_m {
+        font-size: 16px;
+    }
+    &__reaction-button_size_l {
+        font-size: 16px;
+    }
+    &__reaction-button_size_xl {
+        font-size: 20px;
+    }
+
+    &__reaction-button-text_size_xs {
+        font-size: 10px;
+    }
+    &__reaction-button-text_size_s {
+        font-size: 12px;
```

Comment:
I guess font-size should be taken from uikit tokens

Reply:
You mean css-variables like `--g-text-display-1-font-size`?

Reply:
Yes

Reply:
Used these font-sizes:
`xs`: `--g-text-caption-1-font-size`
`s`: `--g-text-caption-2-font-size`
`m`: `--g-text-body-1-font-size`
`l`: `--g-text-subheader-1-font-size`
`xl`: `--g-text-subheader-2-font-size`